cms/src/main/java/uk/nhs/digital/JcrQueryHelper.java

```diff
@@ -0,0 +1,39 @@
+package uk.nhs.digital;
+
+import static java.text.MessageFormat.format;
+
+import org.apache.jackrabbit.util.Text;
+import org.hippoecm.repository.HippoStdNodeType;
+
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import javax.jcr.Session;
+import javax.jcr.query.Query;
+import javax.jcr.query.QueryResult;
+
+public class JcrQueryHelper {
+
+    public static QueryResult executeSql2Query(Session session, String sql2QueryTemplate, String... queryArgs) throws RepositoryException {
```

Comment:
Best to make private - isn't used anywhere outside of this class and hiding it this way will prevent its use with unescaped values.